pdk/1.0/modules/ROOT/pages/policies-pdk-create-project.adoc

```diff
@@ -21,7 +21,7 @@ Before you can create a custom policy project, ensure that you have installed al
 [[create-a-new-policy-project]]
 == Create a New Policy Project
 
-To create a custom policy project, run the PDK CLI `create` command, replacing `<my-custom-policy>` with the name of your new policy:
+To create a custom policy project, run the PDK CLI `make create` command, replacing `<my-custom-policy>` with the name of your new policy:
```

Comment:
Uhm in this case its not referring to a makefile goal.

It is referencing the `anypoint-cli-v4 pdk policy-project create`. So, the "make" part at least, shouldn't be there.